tests/performance_tests/performance_test_basic.py

```diff
@@ -199,14 +198,11 @@ def __init__(self, testHelperConfig: TestHelperConfig=TestHelperConfig(), cluste
 
         # Setup cluster and its wallet manager
         self.walletMgr=WalletMgr(True)
-        self.cluster=Cluster(walletd=True, loggingLevel=self.clusterConfig.loggingLevel, loggingLevelDict=self.clusterConfig.loggingDict,
-                             nodeosVers=self.clusterConfig.nodeosVers,unshared=self.testHelperConfig.unshared)
+        self.cluster=Cluster(loggingLevel=self.clusterConfig.loggingLevel, loggingLevelDict=self.clusterConfig.loggingDict,
+                             nodeosVers=self.clusterConfig.nodeosVers,unshared=self.testHelperConfig.unshared,
+                             keepRunning=self.testHelperConfig.dontKill, keepLogs=self.testHelperConfig.keepLogs)
```

Comment:
Looks like `dontKill` and `keepLogs` should move moved out of `TestHelperConfig` and into `ClusterConfig` now.

Reply:
Moved.

Reply:
1739c377d56